design/labels.go

```diff
@@ -10,7 +10,7 @@ var label = a.Type("Label", func() {
 	a.Attribute("type", d.String, func() {
 		a.Enum("labels")
 	})
-	a.Attribute("labelID", d.UUID, "ID of label", func() {
+	a.Attribute("id", d.UUID, "ID of label", func() {
```

Comment:
You only need to revert this one. And it must not be `labelID`. This is the only place that UI deals with.

Reply:
Yes, this is the only change in design spec. I have retained other things as `labelID`

Reply:
Other changes in this PR is required as this change will reflect on `app.Label` object.